### PR TITLE
Website mockup generator (RESUBMISSION)

### PR DESCRIPTION
**Pull request by hackyguru**
Hey, I've corrected based on your suggestions. Thanks!

### CODE REVIEW

File: workshops/website_mockup_generator/README.md [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+
+## What's next? 🚀
+
+![Thinking man](https://media.giphy.com/media/d3mlE7uhX8KFgEmY/giphy.gif)
+
+You are free to customize and contribute to the website mockup generator! You can consider trying out the below if you are interested :
+
+- ![A retro style mockup generator](https://repl.it/@19EUCS071KUMARA/Website-Mockup-Generator)
+- ![A minimalistic mockup generator](https://repl.it/join/dlpvaodr-19eucs071kumara)
+- ![A dark themed mockup generator](https://repl.it/@19EUCS071KUMARA/Website-Mockup-Generator-1)
```
Comment (faisalsayed10) [inline workshops/website_mockup_generator/README.md:483]: no idea why, but images aren't rendering

Comment (hackyguru) [reply workshops/website_mockup_generator/README.md]: yeah fixed them..they're actually links not pics

**Comment by hackyguru** [comment]
Im ready for a re-review @faisalsayed10

File: workshops/website_mockup_generator/README.md [hunk 2]
```diff
@@ -0,0 +1,485 @@
...
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Website Mockup Generator</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body class="body">
+    <h1>Website Mockup Generator</h1>
+    <input type="text" id="websiteAdress" placeholder='Enter the website'>
```
Comment (faisalsayed10) [inline workshops/website_mockup_generator/README.md:134]: typo again

File: workshops/website_mockup_generator/README.md [hunk 3]
```diff
@@ -0,0 +1,485 @@
...
+```
+
+And that's it! We have completed writing our HTML. Your mockup generator should be looking similar to this :
+
+![Mockup generator with only HTML](https://cloud-cg4k9mn15.vercel.app/0mockupgenerator-html.png)
+
+
+### 2) style.css
+
+The style.css file is used to add styles to our website. You can optionally use your own CSS to make our site look better according to your wish!
```
Comment (faisalsayed10) [inline workshops/website_mockup_generator/README.md:151]:
```suggestion
The style.css file is used to add styles to our website. You can optionally use your own CSS to make your site look better according to your wish!

```

File: workshops/website_mockup_generator/README.md [hunk 4]
```diff
@@ -0,0 +1,485 @@
...
+
+```css
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap");
+* {
+  font-family: "Poppins", sans-serif;
+}
+```
+
+#### Customizing the `<body>` style
+Now, let us customize the `<body>` tag. we will change the background color to and the text color to white. To make the website look better, let us align our items to center using `align-items` and `justify-content`:
```
Comment (faisalsayed10) [inline workshops/website_mockup_generator/README.md:164]: typo in the first sentence

File: workshops/website_mockup_generator/README.md [hunk 5]
```diff
@@ -0,0 +1,485 @@
...
+  align-items: center;
+}
+
+input {
+  padding: 0 10px;
+  outline: none;
+}
+
+/*Website address input*/
+#websiteAdress {
```
Comment (faisalsayed10) [inline workshops/website_mockup_generator/README.md:282]: typo

File: workshops/website_mockup_generator/README.md [hunk 6]
```diff
@@ -0,0 +1,485 @@
...
+Firstly, we will be getting the elements from HTML into JavaSript by using their id using the following code :
+
+```javascript
+const resultImg = document.getElementById("resultImg");
+const Download = document.getElementById("Download");
+```
+
+![Button actions](https://cloud-izk2xul9w.vercel.app/0buttonactions.png)
+
+#### Defining the generateMockup() function
```
Comment (faisalsayed10) [inline workshops/website_mockup_generator/README.md:370]: provide code highlighting to `generateMockup()`

**Review by faisalsayed10 — CHANGES_REQUESTED** [review]
Hey, there's still some grammar/spelling errors in the workshop from the past review, make sure you thoroughly read the whole workshop and fix them all

**Pull request closed without merging**